llarp/iwp/linklayer.hpp

```diff
@@ -7,56 +7,69 @@
 #include <crypto/types.hpp>
 #include <link/server.hpp>
 #include <config/key_manager.hpp>
-
+#include <util/thread/queue.hpp>
```

Comment:
maybe I'm missing something but I don't see this new include being used in this file?

Reply:
you're right.